docs/wiki/User Guide/Configurations.md

```diff
@@ -137,6 +137,7 @@ The following configurations are inherited from the open source Kafka client con
 | demotion.history.retention.time.ms                                | Long	  | N	      | 1209600000	                                                                                                                                                                                                                                                                                                                                                                                                           | The maximum time in milliseconds to retain the demotion history of brokers.	                                                                                                                                                                                                                                                                                                |
 | removal.history.retention.time.ms                                 | Long	  | N	      | 1209600000	                                                                                                                                                                                                                                                                                                                                                                                                           | The maximum time in milliseconds to retain the removal history of brokers.	                                                                                                                                                                                                                                                                                                    |
 | logdir.response.timeout.ms                                        | Long	  | N	      | 10000	                                                                                                                                                                                                                                                                                                                                                                                                               | Timeout in ms for broker logdir to respond	                                                                                                                                                                                                                                                                                                                                    |
+| kafka.request.timeout.ms                                          | Integer | N         | 180000
```

Comment:
Broken table entry -- i.e. where is the documentation?

Reply:
This  issue is resolved in another [PR](https://github.com/linkedin/cruise-control/pull/1460)